### ISSUE TITLE
how can I use lizzie in Ubuntu?

### ISSUE THREAD

**Issue by juanda3111** [featurecat/lizzie#225]
I have Ubuntu 18.04 LTS , then my question is: can I run lizzie directly in linux ? or should I emulate with wine in other to use it . Sorry if it is a silly question but I have never tried lizzie before and so far I download the windows 64bits version, I try to run with wine the .exe but didn't worked and later the .tar , and I'm not sure if is because I made some kind of mistake or if is just a problem with wine.

**Comment by cgibbard** [comment]
Don't use wine. It's a java program, just get the package that's for mac/linux, and run the jar with "java -jar Lizzie.jar". Note that in order for it to work, you will need to download a network file, e.g. from http://zero.sjeng.org/best-network uncompress it (using gunzip or whatever GUI tool you like), and rename it to "network" in the directory where Lizzie is.

**Comment by juanda3111** [comment]
the program finally open , but it get frozen, just says "Leela Zero is loading.... " and does nothing  , I already download the network file and rename it to network.txt, and I left the program running for 10 to 15 min. and does nothing. [bildschirmfoto von 2018-05-18 17-57-32](https://user-images.githubusercontent.com/34226466/40261212-fcab42fc-5ac4-11e8-8028-c4bae0dd12ee.png)

**Comment by juanda3111** [comment]
I download the  lizzie-mac-linux.zip , later I unzip it, and I added the network (I also already changed network.txt to network) , and I typed in the terminal in the extracted files "sudo java -jar Lizzie.jar" in order to run the .jar file, but when the program is opened I just see the board, the picture of the  back side and the "Leela Zero is loading.... " sign, but still does nothing.

**Comment by featurecat** [comment]
no by extract "it" I meant the network. it comes in a gzip format.

**Comment by featurecat** [comment]
how many mb is that file?

**Comment by juanda3111** [comment]
I extracted both , the network and the program file.
the network is  116.7 MB .

**Comment by featurecat** [comment]
hmmm I guess you can try the next version which will be released in a few days. I'm not sure what's causing your problem. is your Java the official oracle Java or the alternative? please use the official one as some users had more luck with that.

**Comment by juanda3111** [comment]
Finally ....... I finally solved my problem, First of all, I must to apologize, my problem was that I misunderstood the steps, Yesterday I did once again the hole process but this time I just followed the README file, of the lizzie-mac-linux.zip file. the and my problem was that I didn't add the "leelaz" file to the Lizzie's directory, (basically, I omitted the first step of the README (https://github.com/featurecat/leela-zero ).
The process is actually really straight forward, just by following the README, but I didn't understood what to do since the beginning, because I didn't read it carefully.

Sorry for taking your time, and again thanks for the help. :)
